apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -298,6 +298,34 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
           );
         }}
       />
+      <Controller
+        name="onlyShowFirstAvailableSlot"
+        control={formMethods.control}
+        render={({ field: { value } }) => {
+          const isChecked = value;
+          console.log("Here", isChecked);
+          return (
+            <SettingsToggle
+              toggleSwitchAtTheEnd={true}
+              labelClassName="text-sm"
+              title={t("limit_booking_only_first_slot")}
+              description={t("limit_booking_only_first_slot_description")}
+              checked={isChecked}
+              onCheckedChange={(active) => {
+                if (active) {
+                  formMethods.setValue("onlyShowFirstAvailableSlot", true);
+                } else {
+                  formMethods.setValue("onlyShowFirstAvailableSlot", false);
+                }
+              }}
```

Comment:
small improvement: directly use the boolean instead of using if/else statement to avoid Tautology

```ts
formMethods.setValue("onlyShowFirstAvailableSlot", active);
```

if active can be null or undefined and formMethods.setValue expect a boolean

```ts
formMethods.setValue("onlyShowFirstAvailableSlot", active ?? false);
```